sass/_terminal.scss

```diff
@@ -1,34 +0,0 @@
-//
```

Comment:
Explanation in [this discussion](https://github.com/AcroMedia/commerce_pos/pull/28#discussion_r109998696).